docs/installation/docker.md

```diff
@@ -13,17 +13,15 @@
 git clone https://github.com/fossasia/open-event-server.git && cd open-event-server
 ```
 
-* craete env file using the following command
+* create env file using the following command
 
 ```sh
   cp .env.example .env
 ```
-   Add  SECRET_KEY in the env file to run properly in production mode . To generate a good secret value run python -c 'import secrets;print(secrets.token_hex())'
```

Comment:
This was needed

Reply:
My apologies added them again , should I remove build  image step because docker-compose up -d automatically builds the images and starts the server

Reply:
Do I need to do change anything further